components/avatars/avatars.css

```diff
@@ -0,0 +1,26 @@
+
+.avatar{
+    width: 4rem;
+    height: 4rem;
+    /* border: solid; */
+    /* border-radius: 50%; */
+}
+.avatar>img{
```

Comment:
Give a space between avatar and >,  > and img. Increases readability.

Reply:
Done